src/kvstore/PartManager.h

```diff
@@ -164,6 +166,9 @@ class MetaServerBasedPartManager : public PartManager, public meta::MetaChangedL
 
      void onPartUpdated(const PartMeta& partMeta) override;
 
+     void onLeaderDist(std::unordered_map<GraphSpaceID,
```

Comment:
Maybe "fetchLeaderInfo" is a better name?

Reply:
done!